crawl-ref/source/dat/descript/de/hints.txt

```diff
@@ -919,7 +919,7 @@ du auch dessen Beschreibung lesen.</tiles>
 %%%%
 HINT_ATTACK_MONSTER_MAGIC
 
-Als Hervorrufer wirst du es jedoch mit Magie behandeln wollen. Im
+Als Konjurant wirst du es jedoch mit Magie behandeln wollen. Im
```

Comment:
Not sure about this sentence. I don't see HINT_ATTACK_MONSTER_MAGIC in:
/crawl-ref/source/dat/descript/hints.txt

Probably this is better:
Als Konjurant solltest du jedoch Magie bevorzugen.

Reply:
I checked. It's there in the English file.

%%%%
HINT_ATTACK_MONSTER_MAGIC

However, as a conjurer you will want to deal with it using magic. If you look at
the help entry for the <w>M</w>emorisation screen you'll find an explanation of
how to do this.
%%%%